autodoc.pl

```diff
@@ -1369,11 +1369,13 @@ ($$$)
             my @base_args = $items[0]->{args}->@*;
             my $base_thread_context = $items[0]->{flags} =~ /T/;
             for (my $i = 1; $i < @items; $i++) {
-                no warnings 'experimental::smartmatch';
                 my $item = $items[$i];
+                my $identical_args_count;
```

Comment:
Should probably initialise this to `= 0` in case there are none, to avoid the warning of "use of uninitialized value in !="

Reply:
I've actually picked your other suggestion and force pushed a new commit.